core/simulations/helpers/createVaultWithCollateral.ts

```diff
@@ -186,4 +201,30 @@ const createVaultWithCollateral = async (collateralType: CollateralType, collate
     return vaultId;
 };
 
+const createVaultWithCollateral = async (collateralType: CollateralType, collateralOwned: BigNumber) => {
+    const collateralConfig = getCollateralConfigByType(collateralType);
+    const [balanceSlot, languageFormat] = await determineBalanceSlot(collateralType);
+    if (balanceSlot && languageFormat) {
+        await setCollateralInWallet(collateralConfig.symbol, collateralOwned);
+    } else {
+        // fallback to setting vat balance and withdrawing it
+        await setAndCheckCollateralInVat(collateralType, collateralOwned);
+        await checkAndWithdrawCollateralFromVat(collateralConfig, collateralOwned);
```

Comment:
Do we still need the fallback for RENBTC?

Reply:
Yes, this is different issue that makes us disable RENBTC